src/goto-symex/goto_symex_state.cpp

```diff
@@ -767,14 +767,16 @@ void goto_symex_statet::print_backtrace(std::ostream &out) const
   }
 }
 
-void goto_symex_statet::add_object(
-  ssa_exprt &ssa,
-  std::size_t l1_index,
+ssa_exprt goto_symex_statet::add_object(
+  symbol_exprt expr,
+  std::function<std::size_t(const irep_idt &)> index_generator,
   const namespacet &ns)
 {
   framet &frame = call_stack().top();
 
+  ssa_exprt ssa = rename_ssa<L0>(ssa_exprt{std::move(expr)}, ns);
```

Comment:
note that the `ssa_exprt` constructor takes a reference so the move will have no effect

Reply:
Indeed, even with #3502. I have removed the `std::move` and change the parameter to `const symbol_exprt &`.